scripts/install_moodle.sh

```diff
@@ -401,7 +401,7 @@ EOF
         fastcgi_buffers 16 16k;
         fastcgi_buffer_size 32k;
         fastcgi_param   SCRIPT_FILENAME \$document_root\$fastcgi_script_name;
-        fastcgi_pass unix:/run/php/php7.0-fpm.sock;
+        fastcgi_pass unix:/run/php/php${PhpVer}-fpm.sock;
```

Comment:
This env var `${PhpVer}` is defined below in line 466, so it'll be an empty string here, which is problematic.

Reply:
Hi, sorry about that! I have moved it into a function in the helper script and updated the associated scripts.

cheers
Darrin